packages/pwa-kit-create-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/pwa-kit-create-app",
```

Comment:
Let's make sure that the Changelog file for our generator is updated too.

Reply:
done.